x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_flyout/alerts_flyout.tsx

```diff
@@ -15,81 +15,102 @@ import {
   EuiTitle,
   EuiText,
   EuiHorizontalRule,
-  EuiFlyoutFooter,
   EuiFlexGroup,
   EuiFlexItem,
-  EuiButton,
+  EuiPagination,
+  EuiProgress,
+  EuiLoadingContent,
 } from '@elastic/eui';
 import { AlertsField, AlertsData } from '../../../../types';
 
-const REASON_LABEL = i18n.translate(
-  'xpack.triggersActionsUI.sections.alertsTable.alertsFlyout.reason',
+const NAME_LABEL = i18n.translate(
+  'xpack.triggersActionsUI.sections.alertsTable.alertsFlyout.name',
   {
-    defaultMessage: 'Reason',
+    defaultMessage: 'Name',
   }
 );
 
-const NEXT_LABEL = i18n.translate(
-  'xpack.triggersActionsUI.sections.alertsTable.alertsFlyout.next',
+const REASON_LABEL = i18n.translate(
+  'xpack.triggersActionsUI.sections.alertsTable.alertsFlyout.reason',
   {
-    defaultMessage: 'Next',
+    defaultMessage: 'Reason',
   }
 );
-const PREVIOUS_LABEL = i18n.translate(
-  'xpack.triggersActionsUI.sections.alertsTable.alertsFlyout.previous',
+
+const PAGINATION_LABEL = i18n.translate(
+  'xpack.triggersActionsUI.sections.alertsTable.alertsFlyout.paginationLabel',
   {
-    defaultMessage: 'Previous',
+    defaultMessage: 'Alert navigation',
   }
 );
 
 interface AlertsFlyoutProps {
   alert: AlertsData;
+  flyoutIndex: number;
+  alertsCount: number;
+  isLoading: boolean;
   onClose: () => void;
-  onPaginateNext: () => void;
-  onPaginatePrevious: () => void;
+  onPaginate: (pageIndex: number) => void;
 }
 export const AlertsFlyout: React.FunctionComponent<AlertsFlyoutProps> = ({
   alert,
+  flyoutIndex,
+  alertsCount,
+  isLoading,
   onClose,
-  onPaginateNext,
-  onPaginatePrevious,
+  onPaginate,
 }: AlertsFlyoutProps) => {
   return (
     <EuiFlyout onClose={onClose} size="s" data-test-subj="alertsFlyout">
+      {isLoading && <EuiProgress size="xs" color="accent" data-test-subj="alertsFlyoutLoading" />}
       <EuiFlyoutHeader hasBorder>
-        <EuiTitle size="m" data-test-subj="alertsFlyoutTitle">
-          <h2>{get(alert, AlertsField.name)}</h2>
-        </EuiTitle>
+        <EuiSpacer size="xl" />
+        <EuiFlexGroup gutterSize="none">
```

Comment:
let's add a fake title to see how it looks

Reply:
Sure I can make it like:
<img width="1326" alt="Screen Shot 2022-05-04 at 1 13 52 PM" src="https://user-images.githubusercontent.com/56682/166742585-640ad495-e0ab-476d-aaf8-e0231f51e02b.png">